src/utils/exerciseServer.ts

```diff
@@ -137,7 +137,18 @@ async function exerciseServerWorker(testDir: string, tsserverPath: string, repla
             "arguments": {
                 "preferences": {
                     "disableLineTextInReferences": true, // Match VS Code (and avoid uninteresting errors)
-                    "includePackageJsonAutoImports": "off" // Handle per-request instead
+                    "includePackageJsonAutoImports": "off", // Handle per-request instead
```

Comment:
This can’t be controlled per-request. It does nothing when `includeExternalModuleExports` is false, so this should just be `"auto"`.

Reply:
But `includeExternalModuleExports` is occasionally `true` on a per-request basis, right?

I think over time we should probably just force this to be auto, and then on. If you want to do it now, I can give it a go.

```suggestion
                    "includePackageJsonAutoImports": "auto", // Handle per-request instead
```

Reply:
Yeah, but presumably the intent (either originally or your intent in updating) is to test more realistic completions scenarios, but even when auto-imports were enabled, considering package.json dependencies was always disabled, and the comment incorrectly implied that it could be enabled per-request. IMO it should have been `auto` from the beginning, and should stay `auto` forever, because that’s the default that editors use.

Reply:
Yeah, that's true - I assume the reason Andrew C made it that way was to speed up completions in general. It can't hurt to try it.